### ISSUE TITLE
Can we use this jar to dynamically add key value pairs through code?

### ISSUE THREAD

**Issue by Sambhaji75** [Diallos/logging-event-layout#1]

**Comment by Diallos** [comment]
I didn't get your question, how do you want to test and when you need to add key values pairs ?

**Comment by Sambhaji75** [comment]
In your example, you have the key value pairs added to the config, I'm wondering if those can be dynamically added to the logs through code in a correct JSON format. Btw, I came across your repo while searching for a way to add custom key/value pairs to the json format logging. Thanks.

**Comment by Diallos** [comment]
Yes, you can use MDC through code  to add dynamically to the logs in a correct JSON format,
import org.slf4j.MDC //for log4j2
MDC.put ("key", "value");
 LOG.info(............);
